### PR TITLE
Added rules trialed in navigation repo

### PR DESCRIPTION
**Pull request by seankoji**

### CODE REVIEW

File: resources/.eslintrc-react.js [hunk 1]
```diff
@@ -34,7 +34,7 @@ module.exports = {
         "react/no-unknown-property": "error",                        // Prevent usage of unknown DOM property (fixable)
         "react/no-unused-prop-types": "error",                       // Prevent definitions of unused prop types
         "react/prefer-es6-class": "off",                             // Enforce ES5 or ES6 class for React Components
-        "react/prefer-stateless-function": "off",                    // Enforce stateless React Components to be written as a pure function
+        "react/prefer-stateless-function": "error",                  // Enforce stateless React Components to be written as a pure function
```
Comment (darrennolan) [inline resources/.eslintrc-react.js:37]: 😃

File: resources/.eslintrc-standards.js [hunk 2]
```diff
@@ -106,7 +106,7 @@ module.exports = {
         "no-useless-escape": "off",                              // disallow unnecessary escape characters
         "no-useless-return": "off",                              // disallow redundant return statements
         "no-void": "off",                                        // disallow void operators
-        "no-warning-comments": "off",                            // disallow specified warning terms in comments
+        "no-warning-comments": "warn",                           // disallow specified warning terms in comments
```
Comment (darrennolan) [inline resources/.eslintrc-standards.js:109]: 😓

**Review by darrennolan — APPROVED** [review]

File: resources/.eslintrc-standards.js (hunk 2 above)
Comment (darrennolan) [reply resources/.eslintrc-standards.js:109]: Totally should error that like a boss xD

Comment (seankoji) [reply resources/.eslintrc-standards.js:109]: That will just leave us with undocumented dev debt `xD`

Comment (darrennolan) [reply resources/.eslintrc-standards.js:109]: Shouldn't have dev debt at all!   Stop making it :D

**Pull request merged**